sycl/doc/extensions/supported/sycl_ext_intel_buffer_location.asciidoc

```diff
@@ -82,7 +82,8 @@ a|
 ```c++
 property::buffer_location
 ``` | The buffer_location property notifies the SYCL device compiler that the given accessor will only ever point to the memory identified by the int template parameter of its instance class.
-It also notifies the SYCL runtime to store the given accessor in that memory. | Yes
+It also notifies the SYCL runtime to store the given accessor in that memory. 
+This property is ignored for devices that don't support ```c++ cl_intel_mem_alloc_buffer_location ``` extension. | Yes
```

Comment:
I don't think we should tie the definition of this SYCL extension to a specific OpenCL extension.  Doing so makes it impossible to implement this extension on another backend.  Can we instead say something like:

> This property is meaningful only when the kernel is submitted to a device that has more than one type of global memory (e.g. an FPGA).  When a kernel using this property is submitted to a different device, the property is ignored.

Or, we could write it this way to be specific to FPGA:

> This property is only meaningful on devices that have the aspect `aspect::accelerator` (i.e. FPGA devices).  When a kernel using this property is submitted to a different device, the property is ignored.

Reply:
I prefer first option. I pushed a new commit with it.